php-translation/symfony-bundle/0.4/config/packages/dev/php_translation.yaml

```diff
@@ -0,0 +1,3 @@
+translation:
+    symfony_profiler: # must be placed in config_dev.yml
```

Comment:
Using `packages/dev` is correct 👍

Reply:
I suggest removing this `must be placed in config_dev.yml` comment. It is more confusing than anything else here